### ISSUE TITLE
search only in the middle

### ISSUE THREAD

**Issue by snickers2k** [cyberbrix/episode_splitter#5]
hey. this script is awesome
but sometimes it gives me an error, " more than 3 black segments found - error"

don't know if you still maintaining it (2 years old). 
but is it possible to search just in the middle ? like within 45%-55% percent?

thank you

**Comment by iurirolho** [comment]
i also have this issue i see you made this post, hopefully something can be done to fix it

**Comment by bryce-hoehn** [comment]
Yeah the script isn't super useful in it's current state since it's hard to distinguish between scene transitions and episode cutoffs, but I think this would make it much more consistent.

**Comment by snickers2k** [comment]
probably something like this get 50% of a video
`input=input.mp4; ffmpeg -ss "$(bc -l <<< "$(ffprobe -loglevel error -of csv=p=0 -show_entries format=duration "$input")*0.5")" -i "$input"`
ffprobe to get the duration
bc to calculate half the duration
ffmpeg to make the thumbnail image

from https://superuser.com/questions/477340/how-to-create-a-thumbnail-from-the-middle-of-a-video-with-ffmpeg

somebody able to make this working with episode_splitter ?
